packages/@coorpacademy-components/src/template/playlist-detail/index.js

```diff
@@ -21,9 +20,8 @@ const PlaylistDetail = (props, context) => {
     coverImages,
     playlistRef,
     description,
-    ongoingCourses,
+    ongoingCoursesAvailable,
     playlistCourses,
-    totalCourses,
     filters,
     sorting,
     onBackClick,
```

Comment:
we can do a check on this to handle disabled or not the back button, something like that:

````
 <ButtonLinkIcon
          faIcon="arrow-left"
          data-name="back-button"
          aria-label="Back"
          onClick={onBackClick}
          className={style.backButton}
          tooltipPlacement="right"
         disabled={isNil(onBackClick)}
        />

````

Reply:
ah Huy, the product prefer to hide the button instead of disbaled it